move/walrus_site/sources/site.move

```diff
@@ -53,13 +65,40 @@ module walrus_site::site {
     public fun new_resource(
         path: String,
         blob_id: u256,
-        blob_hash: u256
+        blob_hash: u256,
+        range_start: Option<u256>,
+        range_end: Option<u256>
```

Comment:
The other option would be to use a ptb and pass here `range: Option<Range>` directly, which is constructed in a `Range::new` function.